diff -up btrfs-progs-0.19/Makefile.josef btrfs-progs-0.19/Makefile
--- btrfs-progs-0.19/Makefile.josef	2011-01-10 10:51:24.244225339 -0500
+++ btrfs-progs-0.19/Makefile	2011-01-10 10:51:01.196224723 -0500
@@ -18,7 +18,7 @@ LIBS=-luuid
 
 progs = btrfsctl mkfs.btrfs btrfs-debug-tree btrfs-show btrfs-vol btrfsck \
 	btrfs \
-	btrfs-map-logical
+	btrfs-map-logical btrfstune btrfs-image
 
 # make C=1 to enable sparse
 ifdef C
