diff -up btrfs-progs-0.20-0.1.20121011git91d9eec/mkfs.c.orig btrfs-progs-0.20-0.1.20121011git91d9eec/mkfs.c
--- btrfs-progs-0.20-0.1.20121011git91d9eec/mkfs.c.orig	2012-10-04 17:35:31.000000000 -0700
+++ btrfs-progs-0.20-0.1.20121011git91d9eec/mkfs.c	2012-10-11 12:00:00.901911975 -0700
@@ -381,7 +381,6 @@ static u64 parse_profile(char *s)
 
 static char *parse_label(char *input)
 {
-	int i;
 	int len = strlen(input);
 
 	if (len >= BTRFS_LABEL_SIZE) {
@@ -389,12 +388,6 @@ static char *parse_label(char *input)
 			BTRFS_LABEL_SIZE - 1);
 		exit(1);
 	}
-	for (i = 0; i < len; i++) {
-		if (input[i] == '/' || input[i] == '\\') {
-			fprintf(stderr, "invalid label %s\n", input);
-			exit(1);
-		}
-	}
 	return strdup(input);
 }
 
