diff -up btrfs-progs-0.20-0.1.20121011git91d9eec/version.sh.orig btrfs-progs-0.20-0.1.20121011git91d9eec/version.sh
--- btrfs-progs-0.20-0.1.20121011git91d9eec/version.sh.orig	2012-10-11 13:58:20.890038087 -0700
+++ btrfs-progs-0.20-0.1.20121011git91d9eec/version.sh	2012-10-11 13:58:31.280036171 -0700
@@ -6,7 +6,7 @@
 # Copyright 2008, Oracle
 # Released under the GNU GPLv2
  
-v="Btrfs v0.19"
+v="Btrfs v0.20-rc1"
 
 which git &> /dev/null
 if [ $? == 0 -a -d .git ]; then
